diff --git a/scripts/oneapi.py b/scripts/oneapi.py
index 77ff7b0..09c828d 100644
--- a/scripts/oneapi.py
+++ b/scripts/oneapi.py
@@ -67,6 +67,8 @@ def prod_publish(target='prod-publish'):
     print(target)
     shell('aws s3 sync --only-show-errors --delete s3://%s/oneAPI/versions/%s s3://spec.oneapi.com/oneAPI/versions/%s'
           % (staging_host,common_conf.oneapi_spec_version, common_conf.oneapi_spec_version))
+    shell('aws s3 cp s3://spec.oneapi.com/oneAPI/versions/%s/oneAPI-spec.pdf s3://spec.oneapi.com/oneAPI/oneAPI-spec.pdf'
+          % (common_conf.oneapi_spec_version))
     shell('aws s3 cp s3://%s/oneAPI/index.html s3://spec.oneapi.com/oneAPI/index.html'
           % (staging_host))
     print('published at http://spec.oneapi.com/')
@@ -74,6 +76,8 @@ def prod_publish(target='prod-publish'):
 def stage_publish(target='stage-publish'):
     print(target)
     shell('aws s3 sync --only-show-errors --delete site s3://%s/oneAPI/versions/%s' % (staging_host,common_conf.oneapi_spec_version))
+    shell('aws s3 cp s3://%s/oneAPI/versions/%s/oneAPI-spec.pdf s3://%s/oneAPI/oneAPI-spec.pdf'
+          % (staging_host, common_conf.oneapi_spec_version, staging_host))
     shell('aws s3 cp site/redirect.html s3://%s/oneAPI/index.html' % staging_host)
     print('published at http://staging.spec.oneapi.com.s3-website-us-west-2.amazonaws.com/')
 
